src/js/utils/__tests__/MesosStateUtil-test.js

```diff
@@ -296,4 +296,130 @@ describe("MesosStateUtil", function() {
       });
     });
   });
+
+  describe("#getHostResourcesByFramework", function() {
+    beforeEach(function() {
+      this.mesosState = {
```

Comment:
🤔 Do we have an existing `/tests/fixtures` file that we use in place of this?

Reply:
I copied this from the same file some tests up. I will get some help here in Hamburg today to understand if I can leverage the _fixtures here.

Reply:
Looking at it again, I don't think we should use fixtures from Integration tests at unit testing.

There are fixtures at `MesosStateWithStory.js`, but they do not apply to my use case. I thought of two options here, both with merits.
1. Leave it like this - because we only need `state.tasks` and the work to create a scenario for state that would be fully complete (and reusable) it is too much for this
2. Create a new fixture `MesosStateMixedTasks.js` and add just the tasks part of the object to it, and use it the same way is used `MesosStateWithStory.js`.

Reply:
Okay, I'm fine leaving it as is :+1: Thanks for clarifying and looking into it.